hack/lib/golang.sh

```diff
@@ -558,6 +560,17 @@ kube::golang::build_binaries() {
     local use_go_build
     local -a targets=()
     local arg
+    
+    # Add any files with those //generate annotations in the array below.
+    readonly BINDATAS=( "${KUBE_ROOT}/test/e2e/framework/gobindata_util.go" )
```

Comment:
hmmmm I don't really think we should expose this as a top-level variable (yet) b/c the functionality is so coupled to the implementation of `generate`.  I'd say punt on this unless you insist....  mostly +0 so leaving it for now, lmk if u think its essential